src/fidesops/api/v1/scope_registry.py

```diff
@@ -45,6 +45,8 @@
 SAAS_CONFIG_DELETE = "saas_config:delete"
 
 USER_CREATE = "user:create"
+USER_READ = "user:read"
+ALL_USERS_READ = "user:read-all"
```

Comment:
From our chat earlier I was under the impression that:
- `USER_READ` would be the scope to signify that the token can read all users in the system
- `CURRENT_USER_READ` will be a scope we can add later for the endpoint returning only the current logged in user's data

Is that about right?

Reply:
Yes, that is correct. That is how the api's are implemented right now.

- `USER_READ` is currently doing what is describe in your comment 
- `CURRENT_USER_READ` will be added along with a new GET route in the future per our conversation